torch/csrc/distributed/c10d/Healthcheck.cpp

```diff
@@ -0,0 +1,127 @@
+#include <sys/socket.h>
```

Comment:
you might need to wrap this in `#ifdef _WIN32`
Or something like:
```
#ifndef _MSC_VER
#include <sys/socket.h>
#endif
```

Reply:
ah -- I shouldn't need this header at all actually, will just remove it